src/portfolios/dto/get-transactions.dto.ts

```diff
@@ -0,0 +1,24 @@
+/* istanbul ignore file */
+
+import { ApiPropertyOptional } from '@nestjs/swagger';
+import { IsOptional, IsString } from 'class-validator';
+
+import { IsTicker } from '~/common/decorators/validation';
+
+export class GetTransactionsDto {
+  @ApiPropertyOptional({
+    description: 'Account address involved in transactions',
+    example: '0x0600000000000000000000000000000000000000000000000000000000000000',
```

Comment:
Addresses should be ss58 encoded. `0x0600...` is a DID. Either the description or example needs to be updated.